src/vmm/src/io_uring/operation/mod.rs

```diff
@@ -51,7 +51,7 @@ pub struct Operation<T> {
     pub(crate) len: Option<u32>,
     flags: u8,
     pub(crate) offset: Option<u64>,
-    user_data: Box<T>,
+    user_data: T,
```

Comment:
From what I understand, the `user_data` field here is just a key into a slab. I think it should be renamed into `data_key`(or smth similar). Also generic `<T>` can be removed from the `Operation<T>` as well, as no longer needed.

Reply:
`user_data` used with `Operation` is the actual data, the key is stored in `Sqe.0.user_data`. I can change that to make things clear.

Reply:
Ok